lib/macho/macho_file.rb

```diff
@@ -64,6 +64,10 @@ def magic64?
       Utils.magic64?(header.magic)
     end
 
+    def cmd_round
```

Comment:
It's a bit of a bike shedding topic I guess, but I'm not entirely happy with the terminology. While what is happening at the end is that the size of a command is rounded to a multiple of a certain number by padding the end with some null bytes, the number being returned here is probably better named `alignment`. That's basically the whole point of padding the load commands, namely to ensure that every load command starts at a memory address that is a multiple of either 4 or 8, because for certain architectures this alignment is important to be able to actually access the individual fields of those load command structures. (I think this is also more recognizable terminology for people familiar with C-like languages.)

Sorry for the long introduction, I just tried to provide enough context. **TL;DR** It would be nice to replace all newly introduced instances of `cmd_round` or `round` with `alignment`.